test/e2e/set/scyllacluster/verify.go

```diff
@@ -116,23 +116,38 @@ func verifyScyllaCluster(ctx context.Context, kubeClient kubernetes.Interface, s
 
 	verifyPersistentVolumeClaims(ctx, kubeClient.CoreV1(), sc)
 
-	// TODO: Use scylla client to check at least "UN"
 	scyllaClient, hosts, err := utils.GetScyllaClient(ctx, kubeClient.CoreV1(), sc)
 	o.Expect(err).NotTo(o.HaveOccurred())
 	defer scyllaClient.Close()
 
 	o.Expect(hosts).To(o.HaveLen(memberCount))
+}
 
-	status, err := scyllaClient.Status(ctx, "")
+func getScyllaHostsAndWaitForFullQuorum(ctx context.Context, client corev1client.CoreV1Interface, sc *scyllav1.ScyllaCluster) []string {
+	framework.By("Waiting for the ScyllaCluster to reach consistency ALL")
```

Comment:
i'd prefer util not to use the framework for separation of concerns and also to be possibly transferable to non e2e utils. I should fix the data inserter as well, since I've moved it after the refactor and it uses some info calls

Reply:
hm, looks like actually already have the framework there, maybe it's the By call that doesn't fit me there too well. I wasn't even eager to hide it into a method but  in this case that just helped a lot